src/main/java/com/indeed/jiraactions/ActionFactory.java

```diff
@@ -174,7 +174,7 @@ private long getDateResolved(final String resolutionDate) {
         if (StringUtils.isEmpty(resolutionDate)) {
             return 0;
         } else {
-            return Long.parseLong(DateTime.parse(resolutionDate).toString("yyyyMMdd"));
+            return Long.parseLong(resolutionDate.substring(0, 10).replaceAll("-", ""));
```

Comment:
Some values of resolutondate dont have the T to display the time (probably for old issues) and so the .split("T") from earlier and DateTime.parse() wont work. This will only get the "yyyy-MM-dd" then remove the lines